addons/mail/static/src/new/composer/composer.js

```diff
@@ -340,6 +340,11 @@ export class Composer extends Component {
             this.attachmentUploader?.clear();
             this.props.composer.textInputContent = "";
             el.focus();
+        } else {
```

Comment:
The issue is that this message will also be displayed when pressing `Enter` on an empty composer. I think this should be an `else if` instead.

Reply:
Yes, else is too generic for this message. I didn't think about the empty message.
Now it test that some attachment are still uploading